process-dataset/steps/upload-file-to-aws.js

```diff
@@ -0,0 +1,27 @@
+const fsPromises = require('fs').promises;
+const upload = require("../../aws");
+
+const uploadFileToS3 = async (id, readFolder, filename) => {
```

Comment:
couldn't this also be used for the `upload features to aws`?

Reply:
yes, it could but I didn't want to refactor things too far away from what they were before